music21/musicxml/m21ToXml.py

```diff
@@ -4823,7 +4893,7 @@ def articulationToXmlTechnical(self, articulationMark):
             musicXMLTechnicalName = 'other-technical'
 
         # TODO: support additional technical marks listed above
-        if musicXMLTechnicalName in ('hammer-on', 'pull-off', 'bend', 'hole', 'arrow'):
+        if musicXMLTechnicalName in ('bend', 'hole', 'arrow'):
```

Comment:
Since this is now unreachable ('bend' etc are not in the dict):

```suggestion
        if musicXMLTechnicalName in ('bend', 'hole', 'arrow'):  # pragma: no cover
```

This will exempt from test coverage. But leaving this in is good in case we add these classes to the dict but don't fully implement in future.